user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Feeds/Specifying_data_input_in_a_dashboard_URL.md

```diff
@@ -97,14 +122,15 @@ To pass this JSON object as part of a URL, it needs to be URL-encoded.
 
 `https://[DMA IP]/dashboard/#/MyDashboards/dashboard.dmadb?data=%7B%22version%22%3A1%2C%22components%22%3A%5B%7B%22cid%22%3A123%2C%22select%22%3A%7B%22elements%22%3A%5B%221%2F2%22%2C%221%2F8%22%2C%22212%2F123%22%5D%2C%22parameters%22%3A%5B%221%2F2%2F3%22%2C%221%2F4%2F6%22%5D%7D%7D%5D%7D`
 
-> [!NOTE]
->
-> - " is encoded as %22.
-> - : is encoded as %3A.
-> - , is encoded as %2C.
-> - [ is encoded as %5B.
-> - ] is encoded as %5D.
-> - / is encoded as %2F.
+#### Example 2: low-code app
+
+The following example URL selects one default element on the initial page. The component ID is "1", and the element ID is "1/6"
+
+```url
+https://<dma>/<app-id>?data={"version":1,"components":[{"cid":1,"select":{"elements":["1/6"]}]}}
+```
+
+This is the encoded equivalent: `https://<dma>/<app-id>?data=%7B%22version%22:1,%22components%22:%5B%7B%22cid%22:1,%22select%22:%7B%22elements%22:%5B%221%2F6%22%5D%7D%5D%7D%7D`
```

Comment:
I changed "%22v%22" to "%22version%22". Is this example correct now, @GillesBara?